services/app-graphql/src/schema.graphql

```diff
@@ -1739,6 +1770,11 @@ type Contract {
     SUBMITTED and RESUBMITTED packages cannot be modified
     """
     status: HealthPlanPackageStatus!
+    """
+    Where the contract is in the review submission flow
+    Options are UNDER_REVIEW, APPROVED, WITHDRAWN
+    """
+    reviewStatus: ReviewStatus!
```

Comment:
I don't think we want this field added to `UnlockedContractType`. Thoughts?

Reply:
I could go either way.

An argument for having it in an unlocked contract is that an unlocked contract could have two review statuses `UNDER_REVIEW` or `WITHDRAWN `. There may be a case where we want to know one of those statuses on an unlocked contract.

Reply:
Good point, I brought this up when I spoke with Kelsey this morning and it doesn't sound like we'll need those statuses for an unlocked contract. Since it's not really known yet, I would suggest not implementing it for approvals and wait until we have a better idea about withdraws, etc

Reply:
I added reviewStatus back to unlockedContract. I spoke with @macrael and he thought we should have this status on unlocked because we're defaulting all contracts to UNDER_REVIEW.